ui/helpers/utils/permission.js

```diff
@@ -49,7 +52,13 @@ const PERMISSION_DESCRIPTIONS = deepFreeze({
   [RestrictedMethods.snap_getBip32PublicKey]: (t, _, permissionValue) =>
     permissionValue.caveats[0].value.map(({ path, curve }) => {
       const baseDescription = {
-        leftIcon: 'fas fa-eye',
+        leftIcon: (
+          <Icon
+            name={ICON_NAMES.EYE}
+            margin={4}
+            color={Color.iconAlternative}
+          />
+        ),
```

Comment:
Would it be possible to add a screenshot here for a visual check?

Reply:
Ah, my bad. This file should not be here. I updated it on wrong branch. It's actually for this [PR](https://github.com/MetaMask/metamask-extension/pull/17800). Screenshot is there as well 😄